homotopy-web/src/model/proof/generators.rs

```diff
@@ -38,11 +44,38 @@ impl GeneratorStyle for GeneratorInfo {
     fn shape(&self) -> Option<style::VertexShape> {
         Some(self.shape.clone().into())
     }
+
+    fn color_point(&self) -> Self::Color {
+        self.color.clone()
+    }
+
+    fn color_wire(&self) -> Self::Color {
```

Comment:
This could move into `homotopy_graphics` and it can be an impl function on `GeneratorStyle.